src/pharmpy/modeling/common.py

```diff
@@ -669,6 +668,12 @@ def filter_dataset(model: Model, expr: str):
     -------
     Model
         Pharmpy model object
+
+    Example
+    -------
+    >>> from pharmpy.modeling import *
+    >>> model = load_example_model('pheno')
```

Comment:
This model doesn't have a DVID column. Could it be filtered on another column? Perhaps APGR, WGT or ID